aws/data_source_aws_eks_cluster.go

```diff
@@ -114,6 +114,11 @@ func dataSourceAwsEksCluster() *schema.Resource {
 							Computed: true,
 							Elem:     &schema.Schema{Type: schema.TypeString},
 						},
+						"public_access_cidrs": {
+							Type:     schema.TypeSet,
+							Optional: true,
```

Comment:
This is not a configurable argument in the data source, so we should use `Computed: true` instead.

```suggestion
							Computed: true,
```